android/filament-android/src/main/java/com/google/android/filament/Stream.java

```diff
@@ -33,13 +33,32 @@
     private long mNativeObject;
     private long mNativeEngine;
 
+    /**
+     * Represents the immutable stream type.
+     *
+     * - NATIVE streams are not synchronized but are copy-free.
+     * - TEXID streams are synchronized, but they are GL-only and incur copies.
+     * - ACQUIRED streams are synchronized, copy-free, and take a release callback.
+     */
+    enum StreamType {
+        NATIVE,
+        TEXID,
+        ACQUIRED,
+    };
+
     Stream(long nativeStream, Engine engine) {
         mNativeObject = nativeStream;
         mNativeEngine = engine.getNativeObject();
     }
 
     /**
      * Use <code>Builder</code> to construct an Stream object instance.
+     *
+     * By default, Stream objects are ACQUIRED and must have external images pushed to them via
+     * {@line #setAcquiredImage}.
+     *
+     * To create a NATIVE or TEXID stream, call one of the <pre>stream</pre> methods
```

Comment:
Replace NATIVE/TEXID by `{@link StreamType#NATIVE NATIVE}` and equivalent for TEXID